scripts/fluvio-cms/src/connectors/mod.rs

```diff
@@ -213,6 +220,7 @@ pub enum DataServiceType {
     Mqtt,
     #[strum(serialize = "DuckDB")]
     Duckdb,
+    Graphite,
```

Comment:
Add here `#[strum(serialize = "Graphite")]`, otherwise it will use #[strum(serialize_all = "UPPERCASE")] which seems noisy for this type of connector